docs/multicast-guide.md

```diff
@@ -137,6 +137,13 @@ Support for Windows and IPv6 will be added in the future.
 Configuration option `multicastInterfaces` is not supported with encap mode.
 Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
 
+### Large number of receiver groups for one node
```

Comment:
"Maximum number of receiver groups on one Node"?

Reply:
updated